### PR TITLE
build.sh: skip docker when installed but not running

### PR DESCRIPTION
**Pull request by dhimmel**
closes https://github.com/manubot/rootstock/issues/368

https://stackoverflow.com/a/55283209/4651668
https://stackoverflow.com/a/18035947/4651668

### CODE REVIEW

File: build/build.sh [hunk 1]
```diff
@@ -35,12 +35,12 @@ pandoc --verbose \
   --defaults=common.yaml \
   --defaults=html.yaml
 
-# Return null if docker command is missing, otherwise return path to docker
-DOCKER_EXISTS="$(command -v docker || true)"
+# Set DOCKER_RUNNING to a non-empty string if docker is running, otherwise null.
+DOCKER_RUNNING="$(docker info &> /dev/null && echo "yes" || true)"
```
Comment (dhimmel) [inline build/build.sh:39]: Parts of this command:

1. `docker info &> /dev/null`: run docker info for exit status only, discarding both stdout and stderr
2. `&& echo "yes"`: if part 1 was a success, print "yes" to std, to set `DOCKER_RUNNING=yes`
3. `|| true` if part 1 returned a failing exit status still return a zero (success) status

**Review by agitter — APPROVED** [review]
Thanks for breaking down the command.  This worked as expected on my system when docker was running and not running.

**Comment by dhimmel** [comment]
> This worked as expected on my system when docker was running and not running.

Which OS?

**Comment by agitter** [comment]
> Which OS?

Git Bash on Windows 10

**Pull request merged**